web/src/components/AuthResults/AuthResults.js

```diff
@@ -13,22 +23,28 @@ export default () => {
     setLastUpdate(new Date().toLocaleTimeString())
   }, [currentUser, userMetadata, isAuthenticated])
 
-  return (
-    <div className="w-full overflow-x-auto overflow-y-auto text-sm max-h-80">
-      <p>Last update {lastUpdate}</p>
-      <code>
-        userMetaData:
-        <pre style={{ margin: 0 }}>{JSON.stringify(userMetadata, null, 2)}</pre>
-      </code>
-      <br />
-      <code>
-        currentUser:
-        <pre style={{ margin: 0 }}>{JSON.stringify(currentUser, null, 2)}</pre>
-      </code>
+  const authData = (userMetadata, currentUser) => {
+    let Metadata = 'userMetadata: ' + JSON.stringify(userMetadata, null, 2) + '\n'
+    let User = 'currentUser: ' + JSON.stringify(currentUser, null, 2)
+    return Metadata + User
```

Comment:
```suggestion
    const metadata = 'userMetadata: ' + JSON.stringify(userMetadata, null, 2) + '\n'
    const user = 'currentUser: ' + JSON.stringify(currentUser, null, 2)
    return metadata + user
```
variables with capital letters first are usually reserved for components